src/main/java/org/team1540/robot2024/RobotContainer.java

```diff
@@ -208,8 +216,17 @@ public RobotContainer() {
 
         // Configure the button bindings
         configureButtonBindings();
+        configureLedBindings();
     }
 
+    private void configureLedBindings() {
+        leds.setFatalPattern(LedPatternFlame::new);
```

Comment:
It might be better if this was a more clear error pattern rather than a flame, since flame just seems for aesthetic purposes